test/langtools/jdk/javadoc/doclet/testSimpleTagInherit/TestSimpleTagInherit.java

```diff
@@ -35,16 +35,6 @@
 
 public class TestSimpleTagInherit extends JavadocTester {
 
-    //Javadoc arguments.
```

Comment:
This seems like an unrelated change; is it cleanup?

Reply:
Yes, this is a holdover from the bad old days before `JavadocTester`, when tests were set up using fixed arrays like these. Frankly, I'm surprised these two empty arrays escaped notice for so long.